### ISSUE TITLE
Add frame visualization to run_playback_gripper_keyframes

### ISSUE THREAD

**Issue by rcory** [rcory/drake#31]
Visualization should include frames for any specified bodies (i.e.,  brick) as well as the "goal" frame, similar to what we have in simulation, e.g., `run_planar_gripper_rotate_simulation`, which uses the `FrameViz` class. 

Since the MBP is not part of the simulation diagram for keyframe playback, frames need to be generated from the SceneGraph's frame pose vector, which is the output of `MultibodyPositionToGeometryPose`. The `FrameViz` class takes in a vector of `RigidTransform`s, so there will need to be a conversion leaf system that takes in a frame pose vector and outputs a vector of `RigidTransform`s.

Depends on issue #30

**Comment by rcory** [comment]
Resolved by PR #33
